src/static/js/modules/requestHandler.js

```diff
@@ -51,9 +51,10 @@ RequestHandler.prototype.close = function() {
     this._socket.close();
   }
 };
-RequestHandler.prototype._send = function(type, url, requestData, successCallback, errorCallback) {
+RequestHandler.prototype._send = function(type, url, requestData, successCallback, errorCallback, async) {
   successCallback = successCallback || this.EMPTY_CALLBACK;
   errorCallback = errorCallback || this.EMPTY_CALLBACK;
+  async = async == undefined ? true : false;
```

Comment:
If the `async` is `true`, it won't be `undefined`, and then set to false. Your 'else' value should be `async`
Ex: `async = async == undefined ? true : async;`